### ISSUE TITLE
Can't disable builtin ConverseJS

### ISSUE THREAD

**Issue by agates** [JohnXLivingston/peertube-plugin-livechat#13]
**Describe the bug**
Disabling "Use builtin ConverseJS" in plugin settings doesn't disable the setting.

**To Reproduce**
1. Go to plugin settings
2. Uncheck "Use builtin ConverseJS"
3. Save
4. Refresh

**Expected behavior**
The setting should remain disabled

**Desktop (please complete the following information):**
 - OS: Linux, Pop!_OS 20.10
 - Browser: Brave 
 - Version: 1.21.73

**Additional context**
`chat-use-builtin` in the database is `false`, but is somehow still returning `true` to the UI.  This prevents `chat-uri` from being utilized in the iframe.

**Comment by JohnXLivingston** [comment]
Try uninstall and reinstall the plugin. I had similar problem during development. It seems it is a peertube's bug. Tell me if it is okay.

**Comment by agates** [comment]
Thanks for the quick response -- I have done the following with no success so far:
1. Uninstall and reinstall, no restart
2. Uninstall and reinstall, restart peertube
3. Uninstall, restart peertube, reinstall

Is there maybe some cache getting in the way?

**Comment by JohnXLivingston** [comment]
Have you tried in private browser?

**Comment by agates** [comment]
Just did, no luck :/

**Comment by JohnXLivingston** [comment]
Can you try this in your terminal (replacing «your_instance» with your instance url):
``curl 'https://your_instance/api/v1/plugins/peertube-plugin-livechat/public-settings'``

**Comment by JohnXLivingston** [comment]
What is your instance url?

**Comment by JohnXLivingston** [comment]
Oh, ok. It remains checked on the settings page, but it is disabled in video page.

**Comment by agates** [comment]
`curl 'https://noagendatube.com/api/v1/plugins/peertube-plugin-livechat/public-settings'`

(pretty printed)
```json
{
  "publicSettings": {
    "chat-auto-display": null,
    "chat-open-blank": null,
    "chat-only-locals": true,
    "chat-all-lives": null,
    "chat-all-non-lives": null,
    "chat-videos-list": null,
    "chat-use-builtin": true,
    "chat-uri": "https://red.acted",
    "chat-style": null
  }
}
```

**Comment by JohnXLivingston** [comment]
I can reproduce your bug.

**Comment by JohnXLivingston** [comment]
It is a peertube bug: https://github.com/Chocobozzz/PeerTube/issues/3838
For now, you have to uncheck the settings every time you save the plugin's settings. The value on runtime will be ok (to check, use the curl query). It is the settings form that don't handle correctly default checkbox values (when default=true)

**Comment by JohnXLivingston** [comment]
Thanks for the report.

**Comment by agates** [comment]
I thought that might be the case, however even with the setting disabled (as shown in the database), I still get the following error in logs indicating the iframe isn't being set correctly:

```
error[3/6/2021, 7:08:54 PM] Error in controller.
{
  "err": "Error: Builtin chat disabled.\n    at router.get (/data/plugins/node_modules/peertube-plugin-livechat/main.js:150:15)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"
}
```

**Comment by agates** [comment]
Huh, just checked the database and the value was reverted back to true.  Glad I'm not going crazy here :)

**Comment by JohnXLivingston** [comment]
Yes, everytime you submit the settings forms, you have to uncheck the value first. Annoying... If there is no quick fix on the peertube sides, i will release a plugin version with default value to false.

**Comment by JohnXLivingston** [comment]
I commited a temporary fix: https://github.com/JohnXLivingston/peertube-plugin-livechat/commit/5f0708a942f843a7fda8bfd0655047d2ccec1aae
It will be released today.

**Comment by JohnXLivingston** [comment]
Fixed in 4840bd735a016efa4987a0ee2c2c4754b84041ca
The buggy peertube version is no more supported by the plugin.
